frontend/src/container/NewDashboard/DashboardDescription/index.tsx

```diff
@@ -30,104 +71,430 @@ function DashboardDescription(): JSX.Element {
+
+		const updatedDashboard: Dashboard = {
+			...selectedDashboard,
+			data: {
+				...selectedDashboard.data,
+				layout: [
+					{
+						i: id,
+						w: 12,
+						minW: 12,
+						minH: 1,
+						maxH: 1,
+						x: 0,
+						h: 1,
+						y: 0,
+					},
+					...layouts.filter((e) => e.i !== PANEL_TYPES.EMPTY_WIDGET),
+				],
+				panelMap: { ...panelMap, [id]: newRowWidgetMap },
+				widgets: [
+					...(selectedDashboard.data.widgets || []),
+					{
+						id,
+						title: sectionName,
+						description: '',
+						panelTypes: PANEL_GROUP_TYPES.ROW,
+					},
+				],
+			},
+			uuid: selectedDashboard.uuid,
+		};
+
+		updateDashboardMutation.mutate(updatedDashboard, {
+			// eslint-disable-next-line sonarjs/no-identical-functions
+			onSuccess: (updatedDashboard) => {
+				if (updatedDashboard.payload) {
+					if (updatedDashboard.payload.data.layout)
+						setLayouts(sortLayout(updatedDashboard.payload.data.layout));
+					setSelectedDashboard(updatedDashboard.payload);
+					setPanelMap(updatedDashboard.payload?.data?.panelMap || {});
+				}
+
+				featureResponse.refetch();
+				setIsPanelNameModalOpen(false);
+				setSectionName(DEFAULT_ROW_NAME);
+			},
+			// eslint-disable-next-line sonarjs/no-identical-functions
+			onError: () => {
+				notifications.error({
+					message: SOMETHING_WENT_WRONG,
+				});
+			},
+		});
+	}
+
 	return (
 		<Card className="dashboard-description-container">
-			<Row gutter={16}>
-				<Col flex={1} span={9}>
-					<Typography.Title
-						level={4}
-						style={{ padding: 0, margin: 0 }}
-						data-testid="dashboard-landing-name"
-					>
-						{isDashboardLocked && (
-							<Tooltip title="Dashboard Locked" placement="top">
-								<LockFilled /> &nbsp;
-							</Tooltip>
-						)}
-						{title}
-					</Typography.Title>
-					{description && (
-						<Typography
-							className="dashboard-description"
-							data-testid="dashboard-landing-desc"
-						>
-							{description}
-						</Typography>
-					)}
+			<section className="dashboard-breadcrumbs">
+				<Button
+					type="text"
+					icon={<LayoutGrid size={14} />}
+					className="dashboard-btn"
+					onClick={(): void => history.push(ROUTES.ALL_DASHBOARD)}
+				>
+					Dashboard /
+				</Button>
+				<Button
+					type="text"
+					className="id-btn"
+					icon={
+						// eslint-disable-next-line jsx-a11y/img-redundant-alt
+						<img
+							src={image}
+							alt="dashboard-image"
+							style={{ height: '14px', width: '14px' }}
+						/>
+					}
+				>
+					{title}
+				</Button>
+			</section>
+			<section className="dashbord-details">
+				<div className="left-section">
+					<img
+						src={image}
+						alt="dashboard-img"
+						style={{ width: '16px', height: '16px' }}
+					/>
+					<Typography.Text className="dashboard-title">{title}</Typography.Text>
+					{isDashboardLocked && <LockKeyhole size={14} />}
+				</div>
+				<div className="right-section">
+					<DateTimeSelectionV2 showAutoRefresh hideShareModal />
```

Comment:
this mean to have always have auto refresh and hiding shareModal, please confirm once

Reply:
yes we need to always `showAutoRefresh` and `alwaysHideShareModal` in dashboards landing page